dmd-frontend/src/components/Editor.tsx

```diff
@@ -69,19 +105,19 @@ const Editor = ({ className, onSaveFunc }: comProps) => {
               <Typography className={classes.label} variant="h6" noWrap>
                 New paste
               </Typography>
-              <TextArea value={text}
-                      onChange={handleOnChange}
-                      className={classes.textArea}
-                      rowsMin="15"
+              <TextArea id="content"
+                        name= "content"
+                        //value={contentValue}
+                        onChange={handleChangeContentValue}
+                        className={classes.textArea}
+                        rowsMin="15"
               ></TextArea>
             </Grid>
             <Grid item sm={6} xs={12} zeroMinWidth>   
               <Typography className={classes.label} variant="h6" noWrap>
                 Static content
               </Typography>
-              <Box>
-                {text}
-              </Box>
+              <Markdown contentMarkdown={contentValue}></Markdown>     
```

Comment:
We doesn't need preview feature right now.

Reply:
So for now, should I remove this feature?

Reply:
You need to render to a static HTML file instead of preview. Please remove this code or commit to another branch.